### ISSUE TITLE
Status of this?

### ISSUE THREAD

**Issue by dixonge** [turtl/fdroid#1]
Just checking to see if you saw the last update from fdroid here:

[https://f-droid.org/forums/topic/turtl/](url)

**Comment by orthecreedence** [comment]
Yep, seen it. They reached out outside of the forum and told me that because the project uses Xwalk, it's not available for inclusion via their build process. See here:

> You are most likely using gradle, so add `gradle=yes`.
That builds for me with two issues: First, you are using xwalk. Xwalk
is currently not in any trusted maven repo -- upstream issue is
https://crosswalk-project.org/jira/browse/XWALK-5164. Secondly, the
build generates two APK files, so instead of `gradle=yes` you most
likely just need to add `gradle=armv7`or `gradle=x86`.

In other words, it's kind of at the mercy of the people running the xwalk project. There is the alternative, Aptoid, which I'm looking into.
